pkg/lib/bundle/generate_test.go

```diff
@@ -2,11 +2,14 @@ package bundle
 
 import (
 	"fmt"
+	"github.com/operator-framework/operator-registry/pkg/lib/synthesize"
```

Comment:
nit: can you move this down so it can hangout with other `github.com` folks? It looks odd-one-out up here.